README.md

```diff
@@ -75,9 +75,9 @@ Select 1:
 Identify the three places CSS can go, and rank them in terms of specificity:
 
 ```text
-1.
-2.
-3.
+1.external style sheet
+2.internal style sheet
+3.inline style sheet
 ```
```

Comment:
:+1: although we wouldn't really refer to (2) and (3) as "stylesheets." More accurate names would be in style tags (`<style>`) and inline styling, respectively.